community/blog/2020.10-Introducing-message-handling-with-Azure-Service-Bus-message-influence/README.md

```diff
@@ -0,0 +1,130 @@
+# Introducing message handling with Azure Service Bus message influence in v0.5
+
+[Arcus Messaging](https://messaging.arcus-azure.net/) has been around for quite a while now. It was created to remove the huge hurdle of constantly having to write the same boilerplate for processing message queues. 
+On March 27th, 2020 we've [announced Arcus Messaging](https://www.codit.eu/blog/announcing-arcus-messaging-background-jobs/) for the first time which was just the start of a new journey to where we are today.
+
+Starting from v0.4 and v0.5, the [Arcus Messaging](https://github.com/arcus-azure/arcus.messaging) has got an upgrade in how you can interact with messages coming from Azure Service Bus.
+Previous versions already had set up a message handling system where messages could be routed and processed according to contextual information, but it didn't cover all the scenarios that our customers have and we fell short, for example, it didn't provide a way to manually dead-letter a message.
+
+Until now.
+
+## Fallback message handling
+
+The capability to 'fallback' during message handling is also rather new. When a message is received on the message pump but there's not a single registered message handler that can handle the message correctly, the library provides a way to route your message to a fallback message handler.
+This can of course be used when you expect that the received message will not always be handled. For more information on this topic, see our [dedicated docs page](https://messaging.arcus-azure.net/features/message-pumps/customization#fallback-message-handling) that explains this concept more thoroughly.
+
+The fallback message handling is possible more correct when we're talking about dead lettering a message. Fallback message handlers will process the Azure Service Bus message when none other regular message handler was able to correctly process it.
```

Comment:
```suggestion
The fallback message handling can be used to dead-letter messages, for example, so that an operator can have a look at them and decide they were not correctly processed due to a bug or if it's the expected behavior.
```